File: Web/Tushar-sahani_TusharSahani_2125cs1072_2/Patterns/Pattern4.java
package coderspear.pattern;
import java.util.*;
public class Pattern4 {
   public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();
    for(int i=0;i<n;i++)
    {
        for (int j = 0; j < n; j++) {
            if(j>=i){
                
                System.out.print("*\t");
            }
            else{
                System.out.print("\t");
            }
        }
    System.out.println();
    }
    scn.close();
   }
}

File: AR-VR/Shubhipandey21_ShubhiPandey_2125csit1169_2/patterns/pattern4.cpp
#include <iostream>
using namespace std;

// Pattern 4
void pattern4(int n)
{

    int k = 2 * n - 2;


    for (int i = n; i > 0; i--) {

        for (int j = 0; j < n - i; j++)
            cout << "  ";
        k = k - 2;

        for (int j = 0; j < i; j++) {

            cout << "* ";
        }


        cout << endl;
    }
}

int main()
{
    int n = 5;


    pattern4(n);
    return 0;
}
